packages/rn-tester/RNTesterPods.xcodeproj/xcshareddata/xcschemes/RNTester.xcscheme

```diff
@@ -67,7 +67,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
```

Comment:
Is this change intentional? Why is it flipping to release mode?

Reply:
Ah, this one is not intentional. Accidentially committed while testing `release` part of `react-native-xcode.sh` script.